product.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios'
+
+export async function getProductList(page = 2, pageSize = 12, keyword = '') {
```

Comment:
default parameter는 말그대로 디폴트값으로 좋을만한 값을 보통 넣어주시면 좋습니다.
틀렸다는 건 아니고요. 이 부분은 명세에 따라 결정하시면 됩니다.
```suggestion
export async function getProductList(page = 1, pageSize = 10, keyword = '') {
```